docs/installation/zoom-configuration/README.md

```diff
@@ -10,10 +10,11 @@ Zoom supports two authentication methods for your users to connect Mattermost an
 
 1. [Oauth: ](zoom-setup-oauth.md)
    * Each user will need to connect their Zoom account with their Mattermost account before they can use the integration.  When they try to create a meeting for the first time, they will receive a message to connect their account, and will need to click "Approve" on a pop-up web page.
-   * Users can connect their MM/Zoom accounts even if their emails do not match
+   * Users can connect their Mattermost/Zoom accounts even if their emails do not match.
 2. [JWT/Password](zoom-setup-jwt.md)
-   * Users don't have to connect their account to use the integration which makes it easy to get started.  
+   * Users don't have to connect their account to use the integration which makes it easy to get started.
    * Uses JWT to pass security tokens.  This may not be sufficiently secure for some customers.
 
+**Note:** If you've been using Zoom prior to v1.4 you likely have a webhook-type app configured in Zoom. These are not compatible with Zoom v1.4 and you may experience issues with meeting update information not being posted.
```

Comment:
```suggestion
**Upgrading from a Previous version?** If you've been using Zoom prior to v1.4 you likely have a webhook-type app configured in Zoom. These are not compatible with Zoom plugin v1.4 and you may experience issues with the meeting status message information not being updated when a meeting ends.  This is because the Oauth setup will ALSO send notifications to the server and the plugin gets confused, as a result the meeting status message in the channel will not update correctly until your original Zoom webhook-type app is deactivated.
```

Reply:
@aaronrothschild 
> This is because the Oauth setup will ALSO send notifications to the server and the plugin gets confused, as a result the meeting status message in the channel will not update correctly until your original Zoom webhook-type app is deactivated.

This is not exactly correct. Legacy webhook apps are no longer supported by Zoom or us, but they keep working. If they do not remove it, the only problem they will experience is a lot of log errors saying that a request to the webhook endpoint was expecting a json format request and received a different format. No confusion happening.

The OAuth setup does not by itself setup any webhook (there is even a separate section to configure the webhooks). The difference with previous installations is that you can configure the webhooks and associate them with the app you already created (be it JWT or OAuth, it does not matter). You no longer need a separated webhook app (and you can no longer create one if I am not mistaken).